templates/cluster-template-eks-managedmachinepool-launchtemplate.yaml

```diff
@@ -0,0 +1,161 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: dkc-test-lt
+  namespace: capa-system
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+  infrastructureRef:
+    kind: AWSManagedControlPlane
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    name: dkc-test-lt-control-plane
+  controlPlaneRef:
+    kind: AWSManagedControlPlane
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    name: dkc-test-lt-control-plane
+---
+kind: AWSManagedControlPlane
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+metadata:
+  name: dkc-test-lt-control-plane
+  namespace: capa-system
+spec:
+  region: us-west-2
+  eksClusterName: dkc-eks-lt
+  sshKeyName: default
+  version: v1.19
+  roleName: eks-controlplane.cluster-api-provider-aws.sigs.k8s.io
+  iamAuthenticatorConfig:
+    mapRoles:
+      - groups:
+          - system:masters
+        rolearn: arn:aws:iam::707343435239:role/aws-dev_databricks-view-only
```

Comment:
Guessing these need to be removed from the template :smiley:

Reply:
If you could add the envsubst declarations like the other templates that would be great.

Reply:
oops, thanks for catching this! this is not intended and removed now